app-localize-behavior.html

```diff
@@ -194,6 +195,10 @@
     }
   },
 
+  ready() {
+    this.fire('app-localize-behavior-created', this);
```

Comment:
Done. Feel free to improve the wording if you feel there is a better description.